### PR TITLE
Add cl_intel_create_buffer_with_properties and cl_intel_mem_channel_property to XML

### PR DESCRIPTION
**Pull request by vmaksimo**

### CODE REVIEW

**Review by alycm — APPROVED** [review]
LGTM.

**Comment by alycm** [comment]
I went to merge this but it needs rebased now (trivial, just an adjacent line was touched).

File: xml/cl.xml [hunk 1]
```diff
@@ -5657,6 +5666,15 @@ server's OpenCL/api-docs repository.
...
             </require>
         </extension>
+        <extension name="cl_intel_create_buffer_with_properties" supported="opencl">
+            <require>
+                <command name="clCreateBufferWithPropertiesINTEL"/>
+            </require>
+        </extension>
+        <extension name="cl_intel_mem_channel_property" supported="opencl">
+            <require comment="cl_mem_properties_intel">
+                <enum name="CL_MEM_CHANNEL_INTEL"/>
```
Comment (alycm) [inline xml/cl.xml:5676]:
```suggestion
                <enum name="CL_MEM_CHANNEL_INTEL"/>
            </require>
```

Looks like the merge went a little wrong.

Comment (vmaksimo) [reply xml/cl.xml:5676]: Oh, sorry, thank you for noticing!

**Pull request merged**